### PR TITLE
Use dpctl array only with DPNP_OUTPUT_DPCTL=1

### PR DESCRIPTION
**Pull request by densmirn**

### CODE REVIEW

File: dpnp/dpnp_iface_arraycreation.py [hunk 1]
```diff
@@ -200,7 +200,7 @@ def array(x1, dtype=None, copy=True, order='C', subok=False, ndmin=0, like=None)
         # usm_array has no element wise assignment (aka []) and
         # has no "flat" property and
         # "usm_data.copy_from_host" doesn't work with diffrent datatypes
-        return numpy.array(x1, dtype=dtype, copy=copy, order=order, subok=subok, ndmin=ndmin)
+        return call_origin(numpy.array, x1, dtype=dtype, copy=copy, order=order, subok=subok, ndmin=ndmin)
```
Comment (shssf) [inline dpnp/dpnp_iface_arraycreation.py:203]: This prevents to run pytest with doctl container

Comment (densmirn) [reply dpnp/dpnp_iface_arraycreation.py:203]: It seems `dpctl` container is used with `call_origin` because `call_origin` calls `create_output_container`.

Comment (densmirn) [reply dpnp/dpnp_iface_arraycreation.py:203]: Am I not correct?

**Comment by shssf** [comment]
Please see #887

File: dpnp/dpnp_container.py [hunk 2]
```diff
@@ -46,15 +47,18 @@
...
+
+    if not config.__DPNP_OUTPUT_DPCTL__:
+        warnings.warn("\nDPNP: DPCtl found, but DPCtl array unused. Please set DPNP_OUTPUT_DPCTL=1 to use it.\n")
 
 except ImportError:
     """
     No DPCtl data container available
     """
-    config.__DPNP_DPCTL_AVAILABLE__ = False
+    DPNP_DPCTL_AVAILABLE = False
```
Comment (shssf) [inline dpnp/dpnp_container.py:59]: No need this variable

**Pull request closed without merging**